hack/kind.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -o errexit
+
+KIND_BIN="${KIND_BIN:-kind}"
+KIND_CLUSTER_NAME="${KIND_CLUSTER_NAME:-kind}"
```

Comment:
wdyt about defaulting this to frr-k8s? feels less invasive to not occupy the default cluster name :sweat_smile: (+ having a different path for this project's kubeconfig won't hurt)

Reply:
makes sense